src/main/java/com/uber/jenkins/phabricator/conduit/Differential.java

```diff
@@ -119,7 +121,13 @@ public String getSummaryMessage(String phabricatorURL) {
 
     public String getPhabricatorLink(String phabricatorURL) {
         String revisionID = this.getRevisionID(true);
-        return String.format("%s%s", phabricatorURL, revisionID);
+        try {
+            URL base = new URL(phabricatorURL);
+            return new URL(base, revisionID).toString();
+        } catch (MalformedURLException e) {
+            e.printStackTrace();
+            return String.format("%s%s", phabricatorURL, revisionID);
```

Comment:
You have a good point. I have no idea how the 'd' becomes lowercase. The URL is malformed because there's no slash, and I think this fixes that issue, but a lowercase 'd' is not technically a malformation so that wont be detected here. See my post on #27 for the locations of these bad links.